_security/access-control/document-level-security.md

```diff
@@ -36,6 +36,568 @@ This query specifies that for the role to have access to a document, its `genres
+
+**Role A with DLS**: This role is granted to a broad group of users and includes DLS to restrict access to specific documents, as shown in the following permission set:
+
+```
+{
+  "index_permissions": [
+    {
+      "index_patterns": ["example-index"],
+      "dls": "[.. some DLS here ..]",
+      "allowed_actions": ["indices:data/read/search"]
+    }
+  ]
+}
+```
+
+**Role B without DLS:** This role is specifically granted to certain users, such as administrators, and does not include DLS, as shown in the following permission set:
+
+```
+{
+  "index_permissions" : [
+    {
+      "index_patterns" : ["*"],
+      "allowed_actions" : ["indices:data/read/search"]
+    }
+  ]
+}
+```
+{% include copy.html %}
+
+Setting `plugins.security.dfm_empty_overrides_all` to `true` ensures that administrators assigned Role B can override any DLS restrictions imposed by Role A. This allows specific Role B users to access all documents, regardless of the restrictions applied by Role A's DLS restrictions.
+
+#### Example: Search template access
+
+In this example, two roles are defined, one with DLS and another without DLS, granting access to search templates:
+
+**Role A with DLS:**
+
+```
+{
+  "index_permissions": [
+    {
+      "index_patterns": [
+        "example-index"
+      ],
+      "dls": "[.. some DLS here ..]",
+      "allowed_actions": [
+        "indices:data/read/search",
+      ]
+    }
+  ]
+}
+```
+{% include copy.html %}
+
+**Role B, without DLS**, which only grants access to search templates:
+
+```
+{
+  "index_permissions" : [
+    {
+      "index_patterns" : [ "*" ],
+      "allowed_actions" : [ "indices:data/read/search/template" ]---
+layout: default
+title: Document-level security
+parent: Access control
+nav_order: 85
+redirect_from:
+- /security/access-control/document-level-security/
+- /security-plugin/access-control/document-level-security/
+---
+
+# Document-level security (DLS)
+
+Document-level security lets you restrict a role to a subset of documents in an index. The easiest way to get started with document- and field-level security is to open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index permissions** section.
+
+![Document- and field-level security screen in OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/images/security-dls.png)
+
+
+## Simple roles
+
+Document-level security uses the OpenSearch query DSL to define which documents a role grants access to. In OpenSearch Dashboards, choose an index pattern and provide a query in the **Document level security** section:
+
+```json
+{
+  "bool": {
+    "must": {
+      "match": {
+        "genres": "Comedy"
+      }
+    }
+  }
+}
+```
+
+This query specifies that for the role to have access to a document, its `genres` field must include `Comedy`.
```

Comment:
Is this supposed to be repeated?

Reply:
That was an error Stephen thanks for catching.